extensions/windows-event-log/TailEventLog.cpp

```diff
@@ -19,14 +19,16 @@
  */
 
 #include "TailEventLog.h"
+#include <stdio.h>
```

Comment:
I think this, `<queue>` , `<sstream>` and `<iostream>` are unused

Reply:
thanks, fixed in [95aeacd](https://github.com/martinzink/nifi-minifi-cpp/commit/95aeacd78262361de87609058f1f208b01e4fe3b)

Reply:
`<stdio.h>` is still here

Reply:
oh, sorry I had tunnel vision on the highlighted words